### PR TITLE
Morse resource

### PR DESCRIPTION
**Pull request by jaflih**
## Morse Code
- add decode char
- add decode sentence 
- fix linters

### CODE REVIEW

File: morse_code.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+def decode_char(morse)
+  morser = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E', '..-.' => 'F',
+    '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L',
+    '--' => 'M', '-.' => 'N', '---' => 'O', '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R',
+    '...' => 'S', '-' => 'T', '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X',
+    '-.--' => 'Y', '--..' => 'Z'
+  }
+  print morser[morse]
```
Comment (rloterh) [inline morse_code.rb:9]: - [ ] Your app is not fully decoding the full encryption. Kindly ensure to improve upon this aspect.

**Expected output:**
image

**Your output:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Jihane & Mike, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: morse_code.rb (hunk 1 above)
Comment (koachcode) [reply morse_code.rb:9]: Thank you, we will fix it

**Review by FrederickMih — APPROVED** [review]
Hi @jaflih , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by koachcode** [comment]
Thank you @FrederickMih

**Pull request merged**